mbed/CMakeLists.txt

```
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
+set(APP_TARGET mbed-bluetooth-power-consumption-benchmark)
```

That target name makes complete sense but renders the `--flash` option useless:

```
ERROR: Build program file (firmware) not found bluetooth-power-consumption-benchmark/mbed/cmake_build/<target>/develop/<toolchain>/mbed.bin 
```

The solution is to use the name `mbed` 😞

```suggestion
set(APP_TARGET mbed)
```

Seems like a confusion in [mbed-tools](https://github.com/ARMmbed/mbed-tools). I'll look into it...